src/player_hardcoded_effects.cpp

```diff
@@ -885,6 +898,270 @@ void player::hardcoded_effects( effect &it )
             g->events().send<event_type::dies_from_drug_overdose>( getID(), id );
             hp_cur[hp_torso] = 0;
         }
+    } else if( id == effect_hypovolemia ) {
+        // bleed out lambda
+        auto bleed_out = [ this ] {
+            if( has_effect( effect_bleed ) )
+            {
+                add_msg_player_or_npc( m_bad,
+                                       _( "You bleed to death!" ),
+                                       _( "<npcname> bleeds to death!" ) );
+                g->events().send<event_type::dies_from_bleeding>( getID() );
+            } else
+            {
+                add_msg_player_or_npc( m_bad,
+                                       _( "Your heart can't keep up the pace and fails!" ),
+                                       _( "<npcname> has a sudden heart attack!" ) );
+                g->events().send<event_type::dies_from_hypovolemia>( getID() );
+            }
+            hp_cur[hp_torso] = 0;
+        };
+        // this goes first because beyond minimum threshold you just die without delay,
+        // while stage 4 is on a timer check with an rng grace period
+        if( vitamin_get( vitamin_blood ) == -5000 ) {
+            bleed_out();
+        }
+        if( one_in( 900 / intense ) && !in_sleep_state() ) {
+            // Hypovolemic shock
+            // stage 1 - early symptoms include headache, fatigue, weakness, thirst, and dizziness.
+            if( intense == 1 ) {
+                switch( dice( 1, 6 ) ) {
+                    case 1:
+                        add_msg_if_player( m_bad, _( "Your skin looks pale." ) );
+                        break;
+                    case 2:
+                        add_msg_if_player( m_bad, _( "You suddenly feel anxcious for a moment." ) );
+                        break;
+                    case 3:
+                        add_msg_if_player( m_bad, _( "You suddenly feel thirsty." ) );
+                        mod_thirst( -5 );
+                        break;
+                    case 4:
+                        add_msg_if_player( m_bad, _( "You feel dizzy and lightheaded." ) );
+                        add_effect( effect_stunned, rng( 15_seconds, 2_minutes ) );
+                        break;
+                    case 5:
+                        add_msg_if_player( m_bad, _( "You have a headache." ) );
```

Comment:
This fits better with the other messages in my opinion - before, unlike the rest of them, it's not telling you how you feel, just that you have a condition.
That's just my opinion though, feel free to pass this suggestion.
```suggestion
                        add_msg_if_player( m_bad, _( "Your head aches." ) );
```